README.md

```diff
@@ -1,2 +1,26 @@
+
+If labs do make the decision to adopt the NWB standard, they often find themselves motivated to convert their backlogged data to the NWB format. Recognizing the advantages of NWB, labs aim to ensure that their extensive historical datasets align with this standardized framework, enhancing data interoperability, collaboration, and long-term accessibility. However, this effort tends to be a substantial bottleneck in the transition to NWB. The accumulation of historical data, often stored in diverse and proprietary formats, poses a formidable challenge, as researchers must now navigate through and transform these datasets to comply with NWB. The sheer volume of legacy data, combined with its inherent heterogeneity, turns the conversion process into a labor-intensive endeavor. Further hampering adoption efforts is the dearth of individual incentives for researchers to prioritize this time-consuming task. In the fast-paced environment of scientific research, where immediate outcomes and results often take precedence, dedicating time and resources to the retrospective conversion of historical data may not be perceived as an urgent priority for individual researchers. The absence of tangible, career-advancing rewards or incentives further hampers the motivation to address this critical aspect of NWB adoption.
+
+One common approach to this problem is to hire or leverage dedicated research software engineers (RSEs) [cite Ritt Group, NYU, Princeton] or contract with data management/consulting companies (ex. DataJoint) to convert the data. But, since these resources tend to be distributed across many different universities in institutions, much of the work maybe duplicated. Common conversion frameworks and tools can help accelerate development and make conversions easier for all stakeholders.
+
+Just as labs frequently seek to convert backlogged data, they are also often motivated to establish conversion pipelines in advance of experimental data collection, so that newly generated data can be automatically converted into NWB with minimal duplicated work. For example, a lab may develop a conversion pipeline on existing data, and then try to generalize the pipeline so that it can be used for any follow up experiments. Automatically converting diverse neurophysiology data formats presents challenges due to the wide variety of modalities involved. Even a single lab may employ voltage recording, optical imaging, optogenetics, behavior, and more, each with its own software-dependent formats lacking standardization. A lab-specific conversion pipeline must take all the possible options into account to properly future-proof, or adapt itself to every new experiment that adds or removes a piece of software or hardware. These considerations make conversion pipelines using _pyNWB_ or _MatNWB_ alone difficult to develop and even harder to maintain, and are often cited as a specific pain point [cite A perspective on neuroscience data standardization with NWB].
```

Comment:
```suggestion
Just as labs frequently seek to convert backlogged data, they are also often motivated to establish conversion pipelines in advance of experimental data collection, so that newly generated data can be automatically converted into NWB with minimal duplicated work. For example, a lab may develop a conversion pipeline on existing data, and then try to generalize the pipeline so that it can be used for any follow up experiments. Automatically converting diverse neurophysiology data formats presents challenges due to the wide variety of modalities involved. Even a single lab may employ voltage recording, optical imaging, optogenetics, behavior, and more, each with its own software-dependent formats lacking standardization. A lab-specific conversion pipeline must take all the possible options into account to properly future-proof, or adapt itself to every new experiment that adds or removes a piece of software or hardware. These considerations make conversion pipelines using _PyNWB_ or _MatNWB_ alone difficult to develop and even harder to maintain, and are often cited as a specific pain point [cite A perspective on neuroscience data standardization with NWB].
```